tensorflow_federated/python/core/impl/executors/federating_executor_test.py

```diff
@@ -820,8 +807,7 @@ def comp():
       self.assertEqual(v.internal_representation.numpy(), 11)
 
   @parameterized.named_parameters(
-      ('centralized',
-       federating_executor.CentralizedIntrinsicStrategy),
+      ('centralized', federating_executor.CentralizedIntrinsicStrategy),
```

Comment:
hmm, i ran the [`linting command`](https://github.com/tf-encrypted/federated/blob/master/CONTRIBUTING.md#general-guidelines) looks like it changed the format at different places.

@jvmncs did you run this command on your previous PR?

Reply:
I did not -- opened a separate issue to clean this up before submitting a PR upstream